### DISCUSSION TITLE
remote target list input support?

### DISCUSSION THREAD

**Discussion by anquanscan** [projectdiscovery/nuclei#4611] [Q&A] [answered]
Can   -l    support  remote ， -l  http://xxxxx.com/ip.txt,  such as 

nuclei -t http://www.xxx.com/xxx.yaml     -l    http://www.xxx.com/ip.txt

**Comment by peterbenjamin-kh** [comment]
There is a `-turl` / `-template-url` flag for templates (instead of `-t` / `-templates` from your example).

There doesn't seem to be a similar option for `-l` / `-list`

**Comment by ehsandeep** [comment, marked as answer]
@anquanscan no this is not supported.
